cpp/velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -473,6 +473,14 @@ class SubstraitToVeloxPlanConverter {
   /// 'or' expression are effective on the same column.
   static bool childrenFunctionsOnSameField(const ::substrait::Expression_ScalarFunction& function);
 
+  /// Check whether the data type is supported to pushdown.
+  static bool isPushdownSupported(TypePtr inputType);
+
+  /// Check whether the scalar function contains data type that doesn't to pushdown.
+  static bool canPushdownScalarFunction(
```

Comment:
Could we merge it with `canPushdownFunction` which checks if a scalar function can be pushed down.